From c53b40e0dcc250d75a159389e099dd40b64e5468 Mon Sep 17 00:00:00 2001
From: Dan Crowell <dcrowell@us.ibm.com>
Date: Wed, 27 May 2015 16:39:23 -0500
Subject: [PATCH] Remove gard actions for memory plugging errors

Modified errors related to unsupported or invalid dimm
configurations to no longer gard any hardware.

Change-Id: I67459425e973ad1c6f4cf95ccd164b73bdd46a80
---
 .../mss_eff_config/memory_mss_eff_config.xml       | 208 +++++----------------
 .../memory_mss_eff_config_termination.xml          |  69 +------
 2 files changed, 50 insertions(+), 227 deletions(-)

diff --git a/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config.xml b/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config.xml
index ed1de55..c6a2d37 100644
--- a/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config.xml
+++ b/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config.xml
@@ -5,7 +5,7 @@
 <!--                                                                        -->
 <!-- OpenPOWER HostBoot Project                                             -->
 <!--                                                                        -->
-<!-- Contributors Listed Below - COPYRIGHT 2013,2014                        -->
+<!-- Contributors Listed Below - COPYRIGHT 2013,2015                        -->
 <!-- [+] International Business Machines Corp.                              -->
 <!--                                                                        -->
 <!--                                                                        -->
@@ -22,7 +22,7 @@
 <!-- permissions and limitations under the License.                         -->
 <!--                                                                        -->
 <!-- IBM_PROLOG_END_TAG                                                     -->
-<!-- $Id: memory_mss_eff_config.xml,v 1.3 2014/10/20 22:12:59 asaetow Exp $ -->
+<!-- $Id: memory_mss_eff_config.xml,v 1.4 2015-05-27 19:56:31 asaetow Exp $ -->
 <!-- For file ../../ipl/fapi/mss_eff_config.C -->
 <!-- // *! OWNER NAME  : Anuwat Saetow     Email: asaetow@us.ibm.com -->
 <!-- // *! BACKUP NAME : Mark Bellows      Email: bellows@us.ibm.com -->
@@ -47,9 +47,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -71,9 +68,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -98,9 +92,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -125,9 +116,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -169,10 +157,10 @@
     <description>Plug rule violation, one position is empty but other are present
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_0</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_1</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_0</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_1</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_0</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_1</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_0</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_1</ffdc>
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
         <priority>HIGH</priority>
@@ -186,10 +174,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -198,10 +182,10 @@
     <description>Plug rule violation, sides do not match
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_0</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_1</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_0</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_1</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_0</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_1</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_0</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -216,10 +200,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 
@@ -229,10 +209,10 @@
     <description>Plug rule violation,  top and bottom do not match
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_0</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_1</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_0</ffdc>
-    <ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_1</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_0</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_0_1</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_0</ffdc>
+	<ffdc>CUR_DIMM_SPD_VALID_U8ARRAY_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -247,10 +227,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -259,10 +235,10 @@
     <description>Incompatable DRAM generation 
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>DRAM_DEVICE_TYPE_0_0</ffdc>
-    <ffdc>DRAM_DEVICE_TYPE_0_1</ffdc>
-    <ffdc>DRAM_DEVICE_TYPE_1_0</ffdc>
-    <ffdc>DRAM_DEVICE_TYPE_1_1</ffdc>
+	<ffdc>DRAM_DEVICE_TYPE_0_0</ffdc>
+	<ffdc>DRAM_DEVICE_TYPE_0_1</ffdc>
+	<ffdc>DRAM_DEVICE_TYPE_1_0</ffdc>
+	<ffdc>DRAM_DEVICE_TYPE_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -277,10 +253,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -289,10 +261,10 @@
     <description>Incompatable DIMM type
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>MODULE_TYPE_0_0</ffdc>
-    <ffdc>MODULE_TYPE_0_1</ffdc>
-    <ffdc>MODULE_TYPE_1_0</ffdc>
-    <ffdc>MODULE_TYPE_1_1</ffdc>
+	<ffdc>MODULE_TYPE_0_0</ffdc>
+	<ffdc>MODULE_TYPE_0_1</ffdc>
+	<ffdc>MODULE_TYPE_1_0</ffdc>
+	<ffdc>MODULE_TYPE_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -307,10 +279,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -319,10 +287,10 @@
     <description>Incompatable DIMM ranks
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>NUM_RANKS_0_0</ffdc>
-    <ffdc>NUM_RANKS_0_1</ffdc>
-    <ffdc>NUM_RANKS_1_0</ffdc>
-    <ffdc>NUM_RANKS_1_1</ffdc>
+	<ffdc>NUM_RANKS_0_0</ffdc>
+	<ffdc>NUM_RANKS_0_1</ffdc>
+	<ffdc>NUM_RANKS_1_0</ffdc>
+	<ffdc>NUM_RANKS_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -337,10 +305,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -349,10 +313,10 @@
     <description>Incompatable DIMM banks
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>SDRAM_BANKS_0_0</ffdc>
-    <ffdc>SDRAM_BANKS_0_1</ffdc>
-    <ffdc>SDRAM_BANKS_1_0</ffdc>
-    <ffdc>SDRAM_BANKS_1_1</ffdc>
+	<ffdc>SDRAM_BANKS_0_0</ffdc>
+	<ffdc>SDRAM_BANKS_0_1</ffdc>
+	<ffdc>SDRAM_BANKS_1_0</ffdc>
+	<ffdc>SDRAM_BANKS_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -367,10 +331,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -379,10 +339,10 @@
     <description>Incompatable DIMM rows
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>SDRAM_ROWS_0_0</ffdc>
-    <ffdc>SDRAM_ROWS_0_1</ffdc>
-    <ffdc>SDRAM_ROWS_1_0</ffdc>
-    <ffdc>SDRAM_ROWS_1_1</ffdc>
+	<ffdc>SDRAM_ROWS_0_0</ffdc>
+	<ffdc>SDRAM_ROWS_0_1</ffdc>
+	<ffdc>SDRAM_ROWS_1_0</ffdc>
+	<ffdc>SDRAM_ROWS_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -397,10 +357,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -409,10 +365,10 @@
     <description>Incompatable DIMM columns
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>SDRAM_COLS_0_0</ffdc>
-    <ffdc>SDRAM_COLS_0_1</ffdc>
-    <ffdc>SDRAM_COLS_1_0</ffdc>
-    <ffdc>SDRAM_COLS_1_1</ffdc>
+	<ffdc>SDRAM_COLS_0_0</ffdc>
+	<ffdc>SDRAM_COLS_0_1</ffdc>
+	<ffdc>SDRAM_COLS_1_0</ffdc>
+	<ffdc>SDRAM_COLS_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -427,10 +383,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -439,10 +391,10 @@
     <description>Incompatable DRAM primary bus width
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>BUS_WIDTH_0_0</ffdc>
-    <ffdc>BUS_WIDTH_0_1</ffdc>
-    <ffdc>BUS_WIDTH_1_0</ffdc>
-    <ffdc>BUS_WIDTH_1_1</ffdc>
+	<ffdc>BUS_WIDTH_0_0</ffdc>
+	<ffdc>BUS_WIDTH_0_1</ffdc>
+	<ffdc>BUS_WIDTH_1_0</ffdc>
+	<ffdc>BUS_WIDTH_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -457,10 +409,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 
@@ -483,10 +431,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 
@@ -496,10 +440,10 @@
     <description>Incompatable DRAM width
         </description>
     <FFDC>TARGET_MBA</FFDC>
-    <ffdc>DRAM_WIDTH_0_0</ffdc>
-    <ffdc>DRAM_WIDTH_0_1</ffdc>
-    <ffdc>DRAM_WIDTH_1_0</ffdc>
-    <ffdc>DRAM_WIDTH_1_1</ffdc>
+	<ffdc>DRAM_WIDTH_0_0</ffdc>
+	<ffdc>DRAM_WIDTH_0_1</ffdc>
+	<ffdc>DRAM_WIDTH_1_0</ffdc>
+	<ffdc>DRAM_WIDTH_1_1</ffdc>
 
     <callout>
         <procedure>MEMORY_PLUGGING_ERROR</procedure>
@@ -514,10 +458,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -540,10 +480,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -566,10 +502,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -591,10 +523,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -616,10 +544,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -641,10 +565,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -666,10 +586,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -691,10 +607,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -716,10 +628,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -741,10 +649,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -766,10 +670,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -791,10 +691,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -816,10 +712,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -841,10 +733,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -866,10 +754,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-        <priority>HIGH</priority>
-    </gard>
 </hwpError>
 
 
diff --git a/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config_termination.xml b/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config_termination.xml
index 9f3c89c..f04b0a6 100644
--- a/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config_termination.xml
+++ b/src/usr/hwpf/hwp/mc_config/mss_eff_config/memory_mss_eff_config_termination.xml
@@ -5,7 +5,9 @@
 <!--                                                                        -->
 <!-- OpenPOWER HostBoot Project                                             -->
 <!--                                                                        -->
-<!-- COPYRIGHT International Business Machines Corp. 2013,2014              -->
+<!-- Contributors Listed Below - COPYRIGHT 2013,2015                        -->
+<!-- [+] International Business Machines Corp.                              -->
+<!--                                                                        -->
 <!--                                                                        -->
 <!-- Licensed under the Apache License, Version 2.0 (the "License");        -->
 <!-- you may not use this file except in compliance with the License.       -->
@@ -20,7 +22,7 @@
 <!-- permissions and limitations under the License.                         -->
 <!--                                                                        -->
 <!-- IBM_PROLOG_END_TAG                                                     -->
-<!-- $Id: memory_mss_eff_config_termination.xml,v 1.2 2014/04/07 23:02:10 lapietra Exp $ -->
+<!-- $Id: memory_mss_eff_config_termination.xml,v 1.3 2015-05-27 20:02:29 asaetow Exp $ -->
 <!-- For file ../../ipl/fapi/mss_eff_config_termination.C -->
 <!-- // *! OWNER NAME  : Dave Cadigan      Email: dcadiga@us.ibm.com -->
 <!-- // *! BACKUP NAME : Anuwat Saetow     Email: asaetow@us.ibm.com -->
@@ -195,9 +197,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -220,9 +219,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 <!-- *********************************************************************** -->
   <hwpError>
@@ -245,9 +241,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -271,9 +264,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -297,9 +287,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -323,9 +310,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -349,9 +333,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -374,9 +355,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -400,9 +378,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -425,9 +400,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -450,9 +422,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -476,9 +445,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -497,9 +463,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -523,9 +486,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 <!-- *********************************************************************** -->
   <hwpError>
@@ -650,9 +610,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -675,9 +632,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -699,9 +653,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 
@@ -724,9 +675,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -750,9 +698,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -820,9 +765,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 <!-- *********************************************************************** -->
@@ -845,9 +787,6 @@
         <target>TARGET_MBA</target>
     </deconfigure>
 
-    <gard>
-        <target>TARGET_MBA</target>
-    </gard>
 </hwpError>
 
 </hwpErrors>
-- 
2.3.0

